joomla/joomla-cms
# [4.0] Load webcomponents with asset manager

The pull depend from #25775, do not merge before that will be merged.

### Summary of Changes
This will load webcomponents with asset manager.


### Testing Instructions
Apply #25775 patch and apply current patch.
Click around, try create/edit content, all should work as before.


### Expected result
Everything works


### Actual result
Everything works


### Documentation Changes Required
Need to update Asset documentation to describe how to add webcomponent to it.
